### DISCUSSION TITLE
How to add mantine to new version of remix?

### DISCUSSION THREAD

**Discussion by MadadiSaeed** [mantinedev/mantine#3187] [Q&A] [answered]
Hi. Remix now uses `renderToPipeableStream` instead of `renderToString`. How to use mantine with new set up?

**Comment by rtivital** [comment]
React streaming is not supported by emotion, see https://github.com/emotion-js/emotion/issues/2800 for more info

**Reply by MadadiSaeed** [reply, marked as answer]
OK. Thanks for the response.
